src/core/src/main/java/org/apache/jmeter/gui/action/Save.java

```diff
@@ -92,7 +92,7 @@
     private static final int BACKUP_MAX_COUNT = JMeterUtils.getPropDefault(JMX_BACKUP_MAX_COUNT, 10);
 
     // NumberFormat to format version number in backup file names
-    private static final DecimalFormat BACKUP_VERSION_FORMATER = new DecimalFormat("000000"); //$NON-NLS-1$
+    private static final DecimalFormat BACKUP_VERSION_FORMAT = new DecimalFormat("000000"); //$NON-NLS-1$
```

Comment:
This has nothing to do with the description of the issue. I don't think it is needed here and should be done in it's own PR (if at all).

Reply:
true, typo unrelated to PR, removed it